helm/chaos-mesh/values.yaml

```diff
@@ -38,6 +38,10 @@ controllerManager:
   allowedNamespaces: ""
   ignoredNamespaces: ""
 
+  # targetNamespace only works with clusterScoped is false(namespace scoped mode).
+  # It means namespace which will be injected chaos
+  targetNamespace: chaos-testing
```

Comment:
I think we don't need to specify the `targetNamespace`. The effected scoped obviously located at where the chaos-mesh deployed in its namespace.